.github/workflows/deploy-preview.yml

```
@@ -48,19 +56,404 @@ jobs:
+      - name: Build and Deploy API
+        id: deploy
+        env:
+          VERCEL_TOKEN: ${{ secrets.VERCEL_TOKEN }}
+          VERCEL_ORG_ID: ${{ secrets.VERCEL_ORG_ID }}
+          VERCEL_PROJECT_ID: ${{ secrets.VERCEL_API_PROJECT_ID }}
+          DATABASE_URL: ${{ env.DATABASE_URL }}
+          DATABASE_URL_UNPOOLED: ${{ env.DATABASE_URL_UNPOOLED }}
+          NEXT_PUBLIC_WEB_URL: https://${{ env.WEB_ALIAS }}
+          NEXT_PUBLIC_ADMIN_URL: https://${{ env.ADMIN_ALIAS }}
+          MOCK_USER_ID: ${{ secrets.MOCK_USER_ID }}
+        run: |
```

actionlint (1.7.9)

shellcheck reported issue in this script: SC2086:info:1:49: Double quote to prevent globbing and word splitting

(shellcheck)